### ISSUE TITLE
Handling different versions of same dataset

### ISSUE THREAD

**Issue by giovsta** [cactool/cactool#79]
**Is your feature request related to a problem? Please describe.**
My problem is that I cannot add different versions of a dataset (coded by different coders) and merge them together. I understand I can do that manually merging the CSV, but I am not sure how CacTool will know about who coded what

**Describe the solution you'd like**
I would like to be able to merge csv datasets in the app, so that I can do a reliability test with a dataset coded elsewhere

**Additional context**
This may already be possible, but the documentation does not talk about it. Actually I am still not sure how the whole software works because the documents do not go into step-by-step processes

**Comment by leelum** [comment]
Hi @giovsta, 

Cactool gives the initials for who coded each individual case on export (see example image below). So if you all work from the same document, you have a good idea of who coded what. 

![image](https://user-images.githubusercontent.com/11173283/161592093-48c2d03b-c7a4-4777-ad3d-3a2667922563.png).

**On the feature -**
You raise an interesting point. The way the database is set up might make it difficult to merge datasets on Cactool itself. We could implement a system which allows you to assign coding ranges within the dataset, so when creating invite links for coders, it would give access to a specified range (So Coder 1 will be given access to posts between 100 and 199; Coder 2 granted access to posts between 200 and 299). If this would solve your issue?

Documentation is currently ongoing, we did have some initial user documentation written up, but since then we've added a bunch of features and isn't accurate!
